src/translator/byte_array_util.h

```diff
@@ -7,6 +7,8 @@ namespace bergamot {
 AlignedMemory loadFileToMemory(const std::string& path, size_t alignment);
 AlignedMemory getModelMemoryFromConfig(marian::Ptr<marian::Options> options);
 AlignedMemory getShortlistMemoryFromConfig(marian::Ptr<marian::Options> options);
+void getVocabsMemoryFromConfig(marian::Ptr<marian::Options> options,
+                               std::vector<AlignedMemory>& vocabMemorySet);
```

Comment:
@qianqianzhu Could you `s/vocabMemorySet/vocabMemories` or something so we don't confuse it to be an `std::set` from variable name, if it's not too much?

Reply:
I think she is using the word set correctly in the sense that it only has the set of vocabularies i.e. the same vocab for source and target is only loaded once.